.github/workflows/vtadmin_web_unit_tests.yml

```diff
@@ -0,0 +1,34 @@
+name: vtadmin-web unit tests
+
+# In specifying the 'paths' property, we need to include the path to this workflow .yml file. 
+# See https://github.community/t/trigger-a-workflow-on-change-to-the-yml-file-itself/17792/4)
+on:
+  push:
+    paths:
+      - '.github/workflows/vtadmin_web_unit_tests.yml'
+      - 'web/vtadmin/**'
+  pull_request:
+    paths:
+      - '.github/workflows/vtadmin_web_unit_tests.yml'
+      - 'web/vtadmin/**'
+
```

Comment:
We should use GitHub Actions' [`concurrency` feature](https://docs.github.com/en/actions/using-jobs/using-concurrency) here. It allows us to reduce the number of concurrent jobs for the same pull request. Let's say you have workflow `A` for commit `n`, if you push a commit `n+1` before `A` had time to finish, the concurrency feature will cancel `A` for commit `n`, and only run `A` for commit `n+1`.

I suggest the following chunk: (which is similar to other workflows like [here](https://github.com/vitessio/vitess/blob/dbbf84b9eaec42bd0b56c5de9a47b3f6d35cf09b/.github/workflows/cluster_endtoend_12.yml#L5) for example)
```yaml
concurrency:
  group: format('{0}-{1}', ${{ github.ref }}, 'vtadmin-web-unit-tests')
  cancel-in-progress: true
```

The `group` key is made unique by concatenating the branch (`github.ref`) and workflow names.

Reply:
sweeeeeeet